### PR TITLE
Add condition to filter out keyvaults

### PR DESCRIPTION
**Pull request by zc-hmcts**
**Does this PR introduce a breaking change?**

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +121,7 @@ aadIdentityName: <Identity Binding>
 keyVaults:
     <VAULT_NAME>:
       excludeEnvironmentSuffix: true
+      disable: true
```
Comment (adusumillipraveen) [inline README.md:124]:
```suggestion
      disabled: true
```

File: library/templates/v2/_secretcsivolumes.tpl [hunk 2]
```diff
@@ -33,9 +35,11 @@ Mount the Key vaults on /mnt/secrets
 {{- if and $languageValues.keyVaults $languageValues.global.enableKeyVaults (not $languageValues.disableKeyVaults) }}
 volumeMounts:
 {{- range $vault, $info := $languageValues.keyVaults }}
+{{- if not $info.disable }}
```
Comment (adusumillipraveen) [inline library/templates/v2/_secretcsivolumes.tpl:38]:
```suggestion
{{- if not $info.disabled }}
```

File: library/templates/v2/_secretproviderclass.tpl [hunk 3]
```diff
@@ -8,6 +8,7 @@
 {{- $keyVaults := $languageValues.keyVaults -}}
 {{- $root := . -}}
 {{- range $vault, $info := $languageValues.keyVaults }}
+{{- if not $info.disable }}
```
Comment (adusumillipraveen) [inline library/templates/v2/_secretproviderclass.tpl:11]:
```suggestion
{{- if not $info.disabled }}
```

**Review by adusumillipraveen — COMMENTED** [review]
let's change to `disabled` aligning other similar flag.

**Review by timja — APPROVED** [review]

**Pull request merged**